flutter-intellij-community.iml

```diff
@@ -12,10 +12,6 @@
       <sourceFolder url="file://$MODULE_DIR$/third_party/intellij-plugins-dart/testSrc" isTestSource="true" />
       <excludeFolder url="file://$MODULE_DIR$/.pub" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
-      <excludeFolder url="file://$MODULE_DIR$/packages" />
```

Comment:
I made a special commit (Fix IML) to ensure these lines did not get removed. I don't know how they got removed. Are they really necessary?